benchmarks/README.md

```diff
@@ -0,0 +1,99 @@
+# ZK-Kit: Merkle Tree Benchmarks
+
+This project is part of the `zk-kit` repository and includes benchmarking tests for three Merkle Tree implementations:
+
+-   **Incremental Merkle Tree (IMT)** from `@zk-kit/imt`
+-   **Lean Incremental Merkle Tree (LeanIMT)** from `@zk-kit/lean-imt`
+-   **Sparse Merkle Tree (SMT)** from `@zk-kit/smt`
+
+The benchmark compares the performance of each Merkle Tree implementation for operations like adding leaves, generating proofs, and verifying proofs.
+
+## Table of Contents
+
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [Benchmark](#benchmark)
+-   [Results](#results)
+-   [Dependencies](#dependencies)
```

Comment:
nit:
nice but unnecessary.
GitHub web ui automatically adds a button to see the table of contents for all `.md` files :wink: 
https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

![image](https://github.com/user-attachments/assets/42395efe-253a-4cf2-ad01-43d90c51fe4d)